tools/sync_audit.py

```
@@ -0,0 +1,172 @@
+
+PR_REF_RE = re.compile(r" \(#\d+\)$")
+
+
+# ---------------------------------------------------------------------------
+# Helpers
+# ---------------------------------------------------------------------------
+
+def git_log_subjects(worktree: Path) -> list[str]:
+    """Return all commit subjects (no PR refs) for the worktree's current branch."""
+    result = subprocess.run(
+        ["git", "log", "--oneline", "--no-merges"],
```

Ruff (0.15.12)

[error] Starting a process with a partial executable path

(S607)